bundles/org.openhab.binding.mqtt.homeassistant/src/main/feature/feature.xml

```diff
@@ -7,11 +7,24 @@
 		<feature>openhab-runtime-base</feature>
 		<feature>openhab-transport-mqtt</feature>
 		<feature dependency="true">openhab.tp-commons-net</feature>
-		<bundle dependency="true">mvn:org.openhab.osgiify/com.hubspot.jinjava.jinjava/2.7.4</bundle>
-		<bundle dependency="true">mvn:org.openhab.osgiify/com.google.re2j.re2j/1.2</bundle>
-		<bundle dependency="true">mvn:ch.obermuhlner/big-math/2.3.2</bundle>
-		<bundle dependency="true">mvn:com.fasterxml.jackson.datatype/jackson-datatype-jdk8/${jackson.version}</bundle>
-		<bundle dependency="true">mvn:org.openhab.osgiify/com.hubspot.immutables.immutables-exceptions/1.9</bundle>
+		<bundle dependency="true">mvn:org.openhab.osgiify/org.graalvm.llvm.llvm-api/24.2.1</bundle>
```

Comment:
Please also check if LLVM API is required here. Do you need to compile during runtime?